crates/iceberg/src/avro/mod.rs

```diff
@@ -18,3 +18,4 @@
 //! Avro related codes.
 #[allow(dead_code)]
```

Comment:
We still allow dead code here?

Reply:
Yes, avro_schema_to_schema still isn't used.